applications/MedApplication/tests/med_files/cube_sub_subgroups/create_mesh.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+###
+### This file is generated automatically by SALOME v9.6.0 with dump python functionality
+###
+
+import salome
```

Comment:
Are these Salome scripts leftovers? Or maybe just some examples? You aren't planning to add Salome as a dependency, right?

Reply:
those are the batch scripts that can be used to recreate the mesh.

I guess I remove them, dont think they will be of much user later ...

And no, I will never add Salome as dependency

Reply:
removed the scripts